build.sbt

```diff
@@ -35,6 +35,12 @@ val compile = Seq(
   "uk.gov.hmrc" %% s"play-frontend-hmrc-$playVersion" % playFrontendHMRCVersion,
   "uk.gov.hmrc" %% s"crypto-json-$playVersion" % "7.6.0",
   "org.jsoup" % "jsoup" % jsoupVersion,
+  guice,
+  "com.typesafe.play" %% "play-guice" % "2.9.0-M2",
```

Comment:
there is a licensing restriction, so we can use only Pekko actors: https://pekko.apache.org/docs/pekko/current/typed/actors.html